solutions/CF Jury Marks.mdx

```diff
@@ -0,0 +1,57 @@
+---
+id: jury-marks
+source: CF
+title: Jury Marks
+author: Danh Ta Chi Thanh
+---
+
+[Official Editorial](https://codeforces.com/blog/entry/53302)
+
+<LanguageSection>
+
+<CPPSection>
+
+**Time Complexity:** $\mathcal{O}(K^2 \log K)$

```

Comment:
Complexity should be outside

Reply:
Is this time complexity fine?
I also think that O(K*(K*logK + K*N)) is suitable.